capstone/src/main/java/com/google/sps/servlets/Utils.java

```diff
@@ -0,0 +1,94 @@
+package com.google.sps.servlets;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.extensions.appengine.datastore.AppEngineDataStoreFactory;
+import com.google.api.client.extensions.appengine.http.UrlFetchTransport;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.Preconditions;
+import com.google.api.client.util.store.DataStoreFactory;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.CalendarScopes;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Collections;
+import javax.servlet.http.HttpServletRequest;
+
+/** Utility class for JDO persistence, OAuth flow helpers, and others. */
+class Utils {
```

Comment:
Nit for future PR: Can you change this name to CalendarUtils to differentiate it from other potential Utils classes?